### PR TITLE
Fixed Shift Listener

### LINKED ISSUE #64: Shift event needs to be triggered by keydown and not keyup
Follow-up to #53

### PR DESCRIPTION
**Pull request by PotassiumK**
Capts when shift is pressed, unsubscribes when shift is released. Fixes
#64.

### CODE REVIEW

File: pixel.js [hunk 1]
```diff
@@ -132,19 +136,32 @@ export default class PixelPlugin
...
+        return handle;
 
+    }
+
+    //Deals with keyboard button down press
+    subscribeToKeyboardPress()
+    {
+        let handle = (e) =>
+        {
+            const shiftKey = 16;
```
Comment (ahankinson) [inline pixel.js]: constants are by convention named in UPPERCASE.

Comment (PotassiumK) [reply pixel.js]: Constants in Diva are all non-uppercased. Where is the line drawn for adopting the prevailing style? 🚒

Comment (PotassiumK) [reply pixel.js]: (Oh actually I get it, sorry)

**Comment by PotassiumK** [comment]
Updated to allow drawing straight lines when the shift key is pressed. I want to optimize this next week, as there are triple nested `if` statements and similar issues that ought to be resolved.

**Pull request merged**